interface/weno/import_pharmacies.php

```diff
@@ -0,0 +1,106 @@
+<?php
```

Comment:
To clarify, this has nothing to do with the import_pharmacies.php script. This is referring to the above interface/weno/img/progress.gif file.